plugins/scenario/hu.bme.mit.gamma.scenario.model/build.properties

```diff
@@ -0,0 +1,10 @@
+#
+
+bin.includes = .,\
+               model/,\
+               META-INF/,\
+               plugin.xml,\
+               plugin.properties
+jars.compile.order = .
+source.. = src/
+output.. = bin/
```

Comment:
If the `model/composite_metamodel.*` `svg`/`pdf`/`png` files are not relevant anymore, they should be deleted.

Reply:
Same for `scenario_metamodel.*`.